extensions/wikia/CreateNewWiki/js/CommunityBuilderOptIn.js

```diff
@@ -81,7 +88,21 @@ define('ext.createNewWiki.communityBuilderOptIn', [], function() {
 			params.push('submit=true');
 		}
 
-		window.location = 'http://community-builder.wikia.com/community/create?'+params.join('&');
+		track({
+			action: tracker.ACTIONS.CLICK,
+			label: 'use-opt-in'
+		});
+
+		setTimeout(function() {
+			window.location = 'http://community-builder.wikia.com/community/create?'+params.join('&');
+		}, 50);
```

Comment:
Is `50` really better than `0`? I know we've run into this issue in the past (where navigation happens right after tracking) but every time I come across it I have to refresh my memory as to the solution.

Reply:
It's what happens in the core tracker, I just used a smaller number. Not sure it's worth trying to figure out what the right amount is.

Reply:
Cool